.github/workflows/main.yml

```diff
@@ -20,3 +20,9 @@ jobs:
   cxx-interop:
     name: Cxx interop
     uses: apple/swift-nio/.github/workflows/cxx_interop.yml@main
+
+  macos-tests:
+    name: macOS tests
+    uses: apple/swift-nio/.github/workflows/macos_tests.yml@main
+    with:
+      build_scheme: swift-cassandra-client
```

Comment:
Is there a `-Package` suffix missing here?

Reply:
I think it omits that suffix if there's only a single product

Reply:
Yeah that's my understanding. I got these by doing `xcodebuild -list` and manually picking something which looked sensible.

Reply:
Good to know, thanks!